### PR TITLE
Add tests for kubectl plugin

### PR DESCRIPTION
**Pull request by ansd**
Relates to #440.

### CODE REVIEW

File: bin/kubectl-rabbitmq [hunk 1]
```diff
@@ -286,13 +290,21 @@ main() {
...
     "install-cluster-operator")
         shift 1
-        if [[ "$#" -gt 1 ]]; then
+        if [[ "$#" -gt 0 ]]; then
             usage
             exit 1
         fi
-        install_cluster_operator "${1:-}"
+        install_cluster_operator
+        ;;
```
Comment (Zerpet) [inline bin/kubectl-rabbitmq:298]: I think this was used to install the Operator using a custom/relocated image. Are we removing support for that feature?

Comment (ansd) [reply bin/kubectl-rabbitmq:298]: This feature was removed in https://github.com/rabbitmq/cluster-operator/commit/d5444f53103d3bba91050390b907c6222338eb32. If we want that feature, we could add it back after this PR has been merged?

**Review by coro — APPROVED** [review]
LGTM. Thanks for adding in the extra options for helptext - that always annoys me when one is missing. Also thanks for the note about GH Actions CPU and the `--unlimited` flag, it's really good to keep that context in the test.

File: bin/kubectl-rabbitmq (hunk 1 above)
Comment (ansd) [reply bin/kubectl-rabbitmq:298]: @Zerpet, I created https://github.com/rabbitmq/cluster-operator/issues/446 which would require this flag. So, we can add it back when working on that issue.

Comment (Zerpet) [reply bin/kubectl-rabbitmq:298]: Ah! I wasn't aware of d5444f53103d3bba91050390b907c6222338eb32, my bad 🙂 I'm happy to conside re-adding this feature as part of #446.

**Review by Zerpet — APPROVED** [review]

**Pull request merged**